packages/ecmascript/src/Shims.ts

```diff
@@ -16,10 +16,10 @@ function getPadString(sourceLength: number, targetLength: number, padString?: st
 	if (sourceLength > targetLength) {
 		return "";
 	}
-	if (padString != null) {
+	if (padString !== null) {
 		padString = "" + padString;
 	}
-	if (padString == null || !padString.length) {
+	if (padString !== null || !padString.length) {
```

Comment:
I just saw this one, I think there should not be a negation, otherwise we are changing behaviour:
```suggestion
	if (padString === null || !padString.length) {
```

Reply:
Hi @VenelinBakalov, thanks for the catch, code updated